pyiron/sphinx/base.py

```diff
@@ -431,7 +431,7 @@ def load_species_group(self, check_overlap=True, potformat='VASP'):
             else:
                 raise ValueError()
         if not check_overlap:
-            self.input.sphinx.pawPot["species"][-1]["checkOverlap"] = "false"
+            self.input.sphinx.pawPot["species"]["checkOverlap"] = "false"
```

Comment:
No. "checkOverlap" resides in the species group (because it overrides the check on a per PAW pot basis). I believe the original version is correct here.